rotary/rules/polish.smk

```diff
@@ -118,11 +92,11 @@ rule polish_polypolish:
         bwa mem -t {threads} -a {input.contigs} {input.qc_short_r2} > {output.mapping_r2} 2>> {log}
 
         printf "\n\n### Polypolish insert filter ###\n" >> {log}
-        {input.polypolish_filter} --in1 {output.mapping_r1} --in2 {output.mapping_r2} \
+        polypolish filter --in1 {output.mapping_r1} --in2 {output.mapping_r2} \
           --out1 {output.mapping_clean_r1} --out2 {output.mapping_clean_r2} 2>> {log}
 
         printf "\n\n### Polypolish ###\n" >> {log}
-        {input.polypolish} {input.contigs} --debug {output.debug} \
+        polypolish polish --debug {output.debug} {input.contigs}  \
           {output.mapping_clean_r1} {output.mapping_clean_r2} 2>> {log} | 
           seqtk seq -A -l 0 | 
```

Comment:
Polypolish 0.6.0 changed now it modifies sequence header names compared to polypolish 0.5.0: https://github.com/rrwick/Polypolish/releases/tag/v0.6.0 (It used to add `_polypolish` to the ends of all sequences, but now it adds ` polypolish`.)

I think we can eliminate the `awk` code line entirely now. Also, we can add the `-C` flag to `seats seq` to drop the header comments. Sounds good?